addons/account/models/account_bank_statement.py

```diff
@@ -296,43 +296,55 @@ def onchange_journal_id(self):
             st_line.journal_id = self.journal_id
             st_line.currency_id = self.journal_id.currency_id or self.company_id.currency_id
 
-    def _check_balance_end_real_same_as_computed(self):
```

Comment:
you can't remove a method in stable version. Maybe you can reuse it instead of `_check_bank_balance_end_real_same_as_computed` ?